packages/webapp/pages/devcards/[userId].tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import {
+  DevCard,
+  DevCardType,
+} from '@dailydotdev/shared/src/components/profile/devcard';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+
+export async function getStaticPaths(): Promise<GetStaticPathsResult> {
+  return { paths: [], fallback: 'blocking' };
+}
+
+interface DevCardPageProps {
+  userId: string;
+}
+
+export async function getStaticProps({
+  params,
+}: GetStaticPropsContext): Promise<GetStaticPropsResult<DevCardPageProps>> {
+  const { userId } = params;
+
+  if (!userId) {
+    return {
+      notFound: true,
+      revalidate: 60,
```

Comment:
we can probably set revalidate to something more like a day or`false` so its always fresh whey making screenshot.